### PR TITLE
Ender Pearl sound

### LINKED ISSUE #159: Pearl
**Is your feature request related to a problem? Please describe.**
Its not a problem

**Describe the solution you'd like**
When shooting pearl everyone will hear enderman tp sound on arena like a big netowork

**Describe alternatives you've considered**
I dont think there's an alternatives

**Additional context**
Nope.

### PR DESCRIPTION
**Pull request by reussy**
Implements a sound for arena players when the ender pearl is hit on a block.

Closes #159

### CODE REVIEW

File: bedwars-plugin/src/main/java/com/andrei1058/bedwars/listeners/EnderPearlLanded.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+        if (!(e.getEntity().getShooter() instanceof Player)) return;
+
+        Player player = (Player) e.getEntity().getShooter();
+
+        IArena iArena = Arena.getArenaByPlayer(player);
+
+        if (!Arena.isInArena(player) || iArena.isSpectator(player)) return;
+
+        Sound sound = Sound.valueOf(Sounds.getSounds().getYml().getString("ender-pearl-landed.sound"));
```
Comment (andrei1058) [inline bedwars-plugin/src/main/java/com/andrei1058/bedwars/listeners/EnderPearlLanded.java:28]: Just use Sounds.playSound("ender-pearl-landed", iArena.getPlayers())

**Review by andrei1058 — CHANGES_REQUESTED** [review]

**Comment by andrei1058** [comment]
also align with master

**Comment by andrei1058** [comment]
bump

**Comment by reussy** [comment]
> bump

Due problems with my PC I made a new pr
#355

**Pull request closed without merging**